PortalSwift/Sources/PortalSwift/Utils/WebSocketClient.swift

```diff
@@ -190,7 +192,7 @@ public class WebSocketClient: Starscream.WebSocketDelegate {
         event: "connect",
         data: ConnectRequestData(
           address: address,
-          chainId: connect.chainId,
+          chainId: connect.chainId ?? 11_155_111,
```

Comment:
Instead of defaulting to Sepolia here, does it make more sense to throw an error if `connect.chainId` is not provided?